ddtrace/contrib/unittest/patch.py

```diff
@@ -0,0 +1,183 @@
+import inspect
+import os
+import unittest
+
+import ddtrace
+from ddtrace import config
+from ddtrace.constants import SPAN_KIND
+from ddtrace.contrib.unittest.constants import COMPONENT_VALUE
+from ddtrace.contrib.unittest.constants import FRAMEWORK
+from ddtrace.contrib.unittest.constants import KIND
+from ddtrace.ext import SpanTypes
+from ddtrace.ext import test
+from ddtrace.internal.ci_visibility import CIVisibility as _CIVisibility
+from ddtrace.internal.ci_visibility.constants import EVENT_TYPE as _EVENT_TYPE
+from ddtrace.internal.constants import COMPONENT
+from ddtrace.internal.utils.wrappers import unwrap as _u
+from ddtrace.vendor import wrapt
+
+
+# unittest default settings
+config._add(
+    "unittest",
+    dict(
+        _default_service="unittest",
+        operation_name=os.getenv("DD_UNITTEST_OPERATION_NAME", default="unittest.test"),
```

Comment:
We should document this environment variable too. Also Is this configuration set in other apm client libraries?

Reply:
Yes, this is used in `pytest`: [docs link](https://ddtrace.readthedocs.io/en/stable/integrations.html#id183). This should've been addressed now.